### PR TITLE
Using proper laud countdown before cleaning of repository

### PR DESCRIPTION
**Pull request by judovana**
There is couple of
```
echo  "doing osmething terrible in Xs"
sleep X
```

here in log only the `doing osmething terrible in Xs` appear. 

I think it may be better to print out proper dynamic countdown instead of just dark screen. WDYT?

I had put it to commn.sh, but some other place may be better. Also I had used it as example only on two places in prepareWorkspace, where there definitely is more applicable places over scripts.

WDYT?

### CODE REVIEW

File: sbin/common/common.sh [hunk 1]
```diff
@@ -243,3 +243,13 @@ function isFromJdk21LTS() {
...
 }
 
+# waits N second (10 by default), printing countdown every second.
+function verboseSleep() {
+  if [[ -z "${1}" ]] ; then
+    local i=10
+  else
+    local i="${1}"
+  fi
+  while [ $i -gt 0 ] ; do echo -n " $i " && sleep 1s && i=$(($i-1)) ; done && echo " $i"
```
Comment (sxa) [inline sbin/common/common.sh:253]: `sleep 1s` won't work on Solaris so remove the `s`.
```suggestion
  while [ $i -gt 0 ] ; do echo -n " $i " && sleep 1 && i=$(($i-1)) ; done && echo " $i"
```

**Review by sxa — COMMENTED** [review]
My initial reaction is that this is someone unneeded but I won't block it ;-)

File: sbin/common/common.sh (hunk 1 above)
Comment (judovana) [reply sbin/common/common.sh:253]: I actually took the `s` from one of temurin-build script, I was sure that defaul it is second for eternity... Thanx for confirming it :)

File: sbin/common/common.sh [hunk 2]
```diff
@@ -243,3 +243,13 @@ function isFromJdk21LTS() {
   [[ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 21 ]] && [[ $(((BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]-21) % 4)) == 0 ]]
 }
 
+# waits N second (10 by default), printing countdown every second.
```
Comment (karianna) [inline sbin/common/common.sh:246]:
```suggestion
# Waits N seconds (10 by default), printing a countdown every second.
```

**Review by karianna — CHANGES_REQUESTED** [review]

**Review by karianna — APPROVED** [review]

File: sbin/common/common.sh (hunk 1 above)
Comment (sxa) [reply sbin/common/common.sh]: Interesting - I might stick a PR in to change it everywhere else then ;-)

**Review by andrew-m-leonard — APPROVED** [review]

**Comment by judovana** [comment]
ok. two approvers done. What now? Can anybody merge please?

**Comment by judovana** [comment]
ty!

**Pull request merged**